### DISCUSSION TITLE
Make a part of an image to be transparent

### DISCUSSION THREAD

**Discussion by Themrpie** [software-mansion/react-native-svg#2149] [Q&A] [answered]
Hello people. I'm trying to take an image and make a part of it transparent, is it possible with this library? I've been able to create part of the image to not be seen by adding a Mask or a Path, but haven't been able to make it transparent meaning that if there was something below the image I would be able to see it, is it possible? I'm trying to use it for OpenAI API
OpenAI docs says the mask I need to send is:

> An additional image whose fully transparent areas (e.g. where alpha is zero) indicate where image should be edited. Must be a valid PNG file, less than 4MB, and have the same dimensions as image.

Hopefully somebody can help.
Thanks

**Comment by jakex7** [comment, marked as answer]
Hey!  
Yes, you can achieve that using an SVG mask, as demonstrated in this simple example (the red view is under the SVG image and is visible through the mask):
<img width="369" alt="image" src="https://github.com/user-attachments/assets/3b55203e-322f-43a9-a2a7-2aa3ec9a281a">
```tsx
<View style={{backgroundColor: 'red'}}>
  <Svg height="400" width="400">
    <Image
      x="0"
      y="0"
      width="400"
      height="400"
      href={require('../assets/office.jpg')}
      mask="url(#mask)"
    />
    <Defs>
      <Mask id="mask" x="0" y="0" width="400" height="400">
        <Rect x="0" y="0" width="400" height="400" fill="white" />
        <Circle cx="200" cy="150" r="100" fill="black" />
      </Mask>
    </Defs>
  </Svg>
</View>
```
